driver-core/src/main/java/com/datastax/driver/core/NettyUtil.java

```diff
@@ -47,21 +46,12 @@
 
     private static final Class[] EVENT_GROUP_ARGUMENTS = {int.class, ThreadFactory.class};
 
+    private static final String ShadingDetectionString = "io.netty.shadingdetection.ShadingIsBroken";
```

Comment:
for consistency, since this is a static final, can you rename this to `SHADING_DETECTION_STRING`?

Reply:
Sorry to be late on this, but we could also inline the constant altogether to avoid having to name it. Also, I would use something other than "ShadingIsBroken" in the string, since shading will be hopefully be fixed after this :)

Reply:
I'll rename the constant to SHADING_DETECTION_STRING as mentioned and the check string to "io.netty.shadingdetection.ShadingDetection" - (ShadingIsBroken expressed my frustration about all the issues that always occur when using shading instead of class loaders =)).

Inlining the var would produce the line 

```
    private static final boolean SHADED = !"io.netty.shadingdetection.ShadingDetection".equals(String.format("%s.%s.shadingdetection.ShadingDetection", "io", "netty")); 
```
I don't think that this looks nice (a string beginning with !) and is a little bit less expressive then using the constant, but if you want I can inline it, too.. Just give me feedback here =)